llvm/include/llvm/IR/PassManager.h

```diff
@@ -836,6 +848,8 @@ class ModuleToFunctionPassAdaptor
 
   static bool isRequired() { return true; }
 
+  void eraseIf(function_ref<bool(StringRef)> Pred) { Pass->eraseIf(Pred); }
```

Comment:
What is this for exactly? I'd hope we don't have to prune out unwanted pre-added passes

Reply:
+1